### PR TITLE
[WIP] - Cloud Instance ownership test cases

### PR DESCRIPTION
**Pull request by rananda**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 362b6c65a906d68e808e7142d50174c1595eb8e9:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/tests/cloud/test_instance_ownership.py [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+                email='abc@redhat.com',
+                group=group_only_user_owned,
+                cost_center='Workload',
+                value_assign='Database')
+    user.create()
+    return user
+
+
+def test_form_button_validation(request, user1, setup_cloud_provider):
+    set_instance_to_user = EC2Instance('cu-9-5', setup_cloud_provider)
```
Comment (psav) [inline cfme/tests/cloud/test_instance_ownership.py]: ```cu-9-5``` should not be hardcoded and should be a yaml entry

File: cfme/tests/cloud/test_instance_ownership.py [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+    login.login(user3.credential.principal, user3.credential.secret)
+    assert(set_instance_to_group.does_vm_exist_in_cfme(), "Instance not found")
+    login.login_admin()
+    # Unset the ownership
+    set_instance_to_group.unset_ownership()
+    login.login(user3.credential.principal, user3.credential.secret)
+    assert(not set_instance_to_group.does_vm_exist_in_cfme(), "Instance exists")
+
+
+@pytest.mark.skipif('True')
```
Comment (psav) [inline cfme/tests/cloud/test_instance_ownership.py]: This shouldn't be in here, we should never just skip a test because with no reason

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by psav** [comment]
@rananda and I talked about this PR, we will probably merge the functionality into a "common" test instead which will address both cloud and infra and will also make use of new User CM object

**Comment by psav** [comment]
Closing this PR off in favour of the common approach

**Pull request closed without merging**